lib/command/update.rb

```diff
@@ -162,91 +162,93 @@ def execute(argv)
 
       interval = Interval.new(@options["interval"])
 
-      update_log = $stdout.capture(quiet: false) do
-        sort_by_key(sort_key, update_target_list).each_with_index do |target, i|
-          display_message = nil
-          data = Downloader.get_data_by_target(target)
-          if !data
-            display_message = "<bold><red>[ERROR]</red></bold> #{target} は管理小説の中に存在しません".termcolor
-          elsif Narou.novel_frozen?(target) && !@options["force"]
-            if argv.length > 0
-              display_message = "ID:#{data["id"]}　#{data["title"]} は凍結中です"
-            else
+      begin
+        update_log = $stdout.capture(quiet: false) do
+          sort_by_key(sort_key, update_target_list).each_with_index do |target, i|
+            display_message = nil
+            data = Downloader.get_data_by_target(target)
+            if !data
+              display_message = "<bold><red>[ERROR]</red></bold> #{target} は管理小説の中に存在しません".termcolor
+            elsif Narou.novel_frozen?(target) && !@options["force"]
+              if argv.length > 0
```

Comment:
__[rubocop]__
* Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)
* Style/ZeroLengthPredicate: Use `!empty?` instead of `length > 0`.

<!-- Comment by [SideCI](https://sideci.com) -->